### PR TITLE
Update managedserviceidentity type enum

### PR DESCRIPTION
**Pull request by Big-Laundry**
Enum is currently missing the ua + sa combination + "none" type. We do support those values in our API and as such need the swagger to reflect that.

### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by adxsdknet** [comment]
# Automation for azure-sdk-for-net 
A PR has been created for you: 
https://github.com/azure/azure-sdk-for-net/pull/5668 
.NET SDK Commits: 
https://github.com/adxsdknet/azure-sdk-for-net/pull/5668/commits/089877dd4db9d93259d8733d745e742ecf157ee6

File: specification/web/resource-manager/Microsoft.Web/stable/2018-02-01/CommonDefinitions.json [hunk 1]
```diff
@@ -1308,7 +1308,9 @@
           "description": "Type of managed service identity.",
           "enum": [
             "SystemAssigned",
-            "UserAssigned"
+            "UserAssigned",
+            "SystemAssigned, UserAssigned",
```
Comment (dsgouda) [inline specification/web/resource-manager/Microsoft.Web/stable/2018-02-01/CommonDefinitions.json]: Why is `UserAssigned` repeated?

**Review by dsgouda — CHANGES_REQUESTED** [review]

File: specification/web/resource-manager/Microsoft.Web/stable/2018-02-01/CommonDefinitions.json (hunk 1 above)
Comment (Hamster-Huey) [reply specification/web/resource-manager/Microsoft.Web/stable/2018-02-01/CommonDefinitions.json]: The values "UserAssigned" and "SystemAssigned, UserAssigned" are distinct enum values. You can see another example on Compute's swagger here: https://github.com/Azure/azure-rest-api-specs/blob/master/specification/compute/resource-manager/Microsoft.Compute/stable/2018-10-01/compute.json#L5867

**Review by dsgouda — APPROVED** [review]
LGTM

**Pull request merged**